### PR TITLE
send reminder to candidate whose interview is sheduled twomorrow

### PR DESCRIPTION
**Pull request by nodeexcel**

### CODE REVIEW

File: src/service/cron.js [hunk 1]
```diff
@@ -10,6 +11,13 @@ export class CronController {
                 });
         }, null, true);
     }
+
+    reminder(email) {
+        new CronJob('00 00 12 * * 1-7', function() {
```
Comment (upexcel) [inline src/service/cron.js]: what is this cron?

File: src/service/reminder.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import mail from '../modules/mail'
+import replaceData from "../modules/replaceVariable";
+import db from "../db";
+
+let reminderMail = (email) => {
+    return new Promise((resolve, reject) => {
+        let dateTime = new Date();
+        let start = moment(dateTime).format("YYYY-MM-DD");
+        let end = moment(start).add(1, 'days').format("YYYY-MM-DD");
+        email.find({ shedule_date: { "$gte": start, "$eq": end } }, { "shedule_date": 1, "shedule_time": 1, "tag_id": 1, "from": 1, "send_template": 1, "sender_mail": 1 }).exec(function(err, response) {
```
Comment (upexcel) [inline src/service/reminder.js]: in db we are storing date in string format? or date time format?

Comment (nodeexcel) [reply src/service/reminder.js]: we are storing in date format

File: src/service/cron.js (hunk 1 above)
Comment (nodeexcel) [reply src/service/cron.js]: sir this cron runs 12 noon every day for sending reminder email to candidate whose email is sheduled to next day

**Pull request merged**